ui/app/src/components/GlobalDialogManager/GlobalDialogManager.tsx

```diff
@@ -102,18 +107,54 @@ function GlobalDialogManager() {
   const versionsBranch = useSelection((state) => state.versions);
   const { enqueueSnackbar } = useSnackbar();
   const dispatch = useDispatch();
+  const { authoringBase } = useEnv();
+  const { username } = useActiveUser();
 
   useEffect(() => {
     const hostToHost$ = getHostToHostBus();
-    const subscription = hostToHost$
-      .pipe(filter((e) => e.type === showSystemNotification.type))
-      .subscribe(({ payload }) => {
-        enqueueSnackbar(payload.message, payload.options);
-      });
+    const subscription = hostToHost$.subscribe(({ type, payload }) => {
+      switch (type) {
+        case showSystemNotification.type:
+          enqueueSnackbar(payload.message, payload.options);
+          break;
+        case newProjectReady.type:
+          const isCreateSiteDialogOpen = Boolean(document.querySelector('[data-dialog-id="create-site-dialog"]'));
+          if (!isCreateSiteDialogOpen && username === payload.user.username) {
+            enqueueSnackbar(
+              <FormattedMessage
+                defaultMessage={`Site "{siteId}" has been created.`}
+                values={{ siteId: payload.siteId }}
+              />,
+              {
+                action: (
+                  <Button
+                    size="small"
+                    onClick={() => {
+                      window.location.href = getSystemLink({
+                        systemLinkId: 'preview',
+                        authoringBase,
+                        site: payload.siteId,
+                        page: '/'
+                      });
+                    }}
+                  >
+                    <FormattedMessage id="words.view" defaultMessage="View" />
+                  </Button>
+                )
+              }
+            );
+          }
+          dispatch(fetchSites());
+          break;
+        case projectDeleted.type:
+          dispatch(fetchSites());
+          break;
```

Comment:
This belongs somewhere else as is has nothing to do with dialogs: `epics/system` perhaps.